CHANGELOG.md

```diff
@@ -27,14 +27,14 @@ Classify the change according to the following categories:
     ### Patches
 
 
-## Develop - 2024-01-07
+## v3.3.0
 ### Minor Updates
+#### Added 
+- Added the following BAU outputs:  lifecycle_chp_standby_cost_after_tax, lifecycle_elecbill_after_tax, lifecycle_production_incentive_after_tax, lifecycle_outage_cost, lifecycle_MG_upgrade_and_fuel_cost
 #### Fixed
 - Fixed setting of default Generator `installed_cost_per_kw` so that user inputs are not overridden 
+- Avoid /summary endpoint error with off-grid runs where there is no ElectricTariff
```

Comment:
Should we add the REopt.jl version update to the Changelog?

Reply:
Yep, done, to v0.39.1